webpack.common.js

```diff
@@ -4,7 +4,16 @@ const { VueLoaderPlugin } = require('vue-loader')
 const NodePolyfillPlugin = require('node-polyfill-webpack-plugin')
 const VuetifyLoaderPlugin = require('vuetify-loader/lib/plugin')
 const path = require('path')
-const deps = require('./package.json').dependencies
+const Dotenv = require('dotenv-webpack')
+const webpack = require('webpack')
+const childProcess = require('child_process')
+let GIT_COMMIT_HASH
+
+try {
+    GIT_COMMIT_HASH = childProcess.execSync('git rev-parse HEAD').toString().trim()
```

Comment:
is it better to use git rev-parse --short HEAD  ? it will return a short hash commit